web/e2e/playwright-tests/tests/login.spec.ts

```
@@ -22,13 +18,27 @@
+      recordVideo: {
+        dir: "videos/",
+        size: { width: 1280, height: 720 }
+      }
+    });
     page = await context.newPage();
     loginPage = new LoginPage(page);
     dashBoardPage = new DashBoardPage(page);
     await page.goto(REEARTH_WEB_E2E_BASEURL, { waitUntil: "networkidle" });
   });
 
+  test.afterEach(async ({}, testInfo) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)